### PR TITLE
Replace emoji codes with characters in README

### PR DESCRIPTION
**Pull request by JimMadge**
## :white_check_mark: Checklist

- [ ] You have given your pull request a meaningful title (_e.g._ `Enable foobar integration` rather than `515 foobar`).
- [ ] You are targeting the appropriate branch. If you're not certain which one this is, it should be **`develop`**.
- [ ] Your branch is up-to-date with the **target branch** (it probably was when you started, but it may have changed since then).

### :arrow_heading_up: Summary

Use emoji characters as the GitHub keys do not render in other places where the README is sourced (_e.g._ PyPI).

### CODE REVIEW

**Review by jemrobinson — APPROVED** [review]
LGTM

**Comment by jemrobinson** [comment]
Is there any other README tidying up we'd like to do in this PR?

**Comment by jemrobinson** [comment]
Can we add a PyPI link and download badge in this PR?

**Comment by jemrobinson** [comment]
Also - if we want to have a different README for PyPI than for GitHub, we can easily do that. Thoughts @JimMadge @craddm ?

**Comment by JimMadge** [comment]
> Also - if we want to have a different README for PyPI than for GitHub, we can easily do that. Thoughts @JimMadge @craddm ?

I'd rather have one if possible. I would guess otherwise we risk the two documents getting out of sync.

**Comment by JimMadge** [comment]
@jemrobinson Do you like the badge with the PyPI version?

We could also do something like 

Static Badge

**Comment by jemrobinson** [comment]
Maybe add the "number of downloads" badge too? Also we could separate/re-order the badges onto different lines, grouping by type e.g. facts (version number, licence, number of contributors), variable statuses (test results, PyPI downloads), ways to contribute (e.g. Slack, email?).

File: README.md [hunk 1]
```diff
@@ -1,31 +1,32 @@
...
 <!-- ALL-CONTRIBUTORS-BADGE:END -->
 
-## :family: Community & support
+## 🧑‍🧑‍🧒 Community & support
 
 - Visit the [Data Safe Haven website](https://data-safe-haven.readthedocs.io) for full documentation and useful links.
 - Join our [Slack server](https://join.slack.com/t/turingdatasafehaven/shared_invite/zt-104oyd8wn-DyOufeaAQFiJDlG5dDGk~w) to ask questions, discuss features, and for general API chat.
 - Open a [discussion on GitHub](https://github.com/alan-turing-institute/data-safe-haven/discussions) for general questions, feature suggestions, and help with our deployment scripts.
 - Look through our [issues on GitHub](https://github.com/alan-turing-institute/data-safe-haven/issues) to see what we're working on and progress towards specific fixes.
 - Subscribe to the [Data Safe Haven newsletter](https://tinyletter.com/turingdatasafehaven) for release announcements.
```
Comment (jemrobinson) [inline README.md]: This isn't available any more. Could perhaps add a link to email the devs?

File: README.md [hunk 2]
```diff
@@ -1,31 +1,32 @@
...
 [![Citation](https://img.shields.io/badge/citation-cite%20this%20project-informational)](https://github.com/alan-turing-institute/data-safe-haven/blob/develop/CITATION.cff)
 <!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
 [![All Contributors](https://img.shields.io/badge/all_contributors-49-orange.svg?style=flat-square)](#contributors-)
 <!-- ALL-CONTRIBUTORS-BADGE:END -->
 
-## :family: Community & support
+## 🧑‍🧑‍🧒 Community & support
 
 - Visit the [Data Safe Haven website](https://data-safe-haven.readthedocs.io) for full documentation and useful links.
 - Join our [Slack server](https://join.slack.com/t/turingdatasafehaven/shared_invite/zt-104oyd8wn-DyOufeaAQFiJDlG5dDGk~w) to ask questions, discuss features, and for general API chat.
```
Comment (jemrobinson) [inline README.md]: Do we also want to open a channel on the UK TRE Slack?

Comment (JimMadge) [reply README.md]: Could do, but I think that is beyond the scope here.

**Pull request merged**